organization/_posts/2025/2025-02-27-coordination.md

```diff
@@ -0,0 +1,192 @@
+
+People are invited to register ASAP so that the local organizers can better prepare for the event. Please note that the early-bird registration deadline is Thursday April 3. Registration closes on April 22.
+
+The timetable has been filled with the different sessions. While the agenda of each session is work in progress, the coarse timetable will not significantly change. This should allow people to start planning for their participation. The event will be an in-person event with (limited) possibility for remote participation. Given the nature of the event (an interactive workshop) we encourage in-person participation.
+
+The HSF organisation team is Michel, Nicole, Eduardo, Paul.
+
+### HSF Seminar Series and Compute Accelerator Forum
+
+[Seminar Indico Category](https://indico.cern.ch/category/18810/).
+[Compute & Accelerator Forum Category](https://indico.cern.ch/category/12741/)
+
+HSF Seminar on 4D reconstruction took place yesterday Feb 26.
+* Very interesting presentations
+* Well attended: 65 connected, 10 in the room. Good sign for the new seminar organization.
+* Meeting recorded, Graeme has it. 
+* As a general point, could engage more non-LHC participation
+    * Next seminar topic may be a good occasion
+
+- 26 March: HSF Seminar - Technical debt on scientific SW (with a focus on some HEP frameworks)
+    - European strategy seminar originally scheduled for this date is postponed
+
+HSF seminar conveners are reachable at hsf-seminar-conveners@gmail.com
+* Please send your suggestions for this Spring seminars
+
+
+### Steering Group
+
+Notes for the [11 February](https://indico.cern.ch/event/1509188/) steering group meeting are linked from indico.
+
+### HSF Affiliated Projects and Software
+
+*We need to find people to help do these reviews.* They are fairly lightweight and should not take a lot of time - we are checking usefulness and impact in the community, as well as software development and project best practices.
+
+As suggested last time a [google spreadsheet](https://docs.google.com/spreadsheets/d/1GCy9Me2mroTdZLtWOdblHyrfkap9BOZp06XkXEd_XYM/edit?gid=0#gid=0) has been set up where people can indicate their willingness to participate as a reviewer in their areas of expertise. Please sign up! And thank you to those who already have. There is also a tab 'Projects' to gather planned and suggested projects for reviews.
+
+
+**[Link to the review template](https://docs.google.com/document/d/1l8Tkruo0EEHwgRBQPmiHq-i4BuwwiR1choRRMUIv6qY/edit?usp=sharing)**
+
+The Steering Group agreed to approach Dirac, Rucio, ACTS, HLS4ML and XrootD to discuss engaging.
+
+The Training Group will offer to help projects who want to improve their tutorials (this is required for a Silver badge).
+
+## Activities Updates
+
+Planning chats between activity conveners and SG liaisons are underway.
+
```

Comment:
Suggest adding:

```css
{:.table .table-hover .table-condensed .table-striped}
```

just above the table. For reasons I do not understand the default CSS used produces terrible results...